packages/emf/backend/sirius-components-emf/src/main/java/org/eclipse/sirius/components/emf/ResourceMetadataAdapter.java

```diff
@@ -47,6 +50,22 @@ public void setName(String name) {
         this.name = name;
     }
 
+    public String getLastMigrationPerformed() {
+        return lastMigrationPerformed;
+    }
+
+    public void setLastMigrationPerformed(String lastMigrationPerformed) {
+        this.lastMigrationPerformed = lastMigrationPerformed;
+    }
+
+    public String getMigrationVersion() {
+        return migrationVersion;
+    }
+
+    public void setMigrationVersion(String migrationVersion) {
+        this.migrationVersion = migrationVersion;
+    }
```

Comment:
You don't need two setters since both information should be updated simultaneously something like this:

```
public void updateMigrationData(String lastMigrationPerformed, String newVersion) {}
```

would be more explicit